mesonbuild/coredata.py

```diff
@@ -367,7 +367,8 @@ def __init__(self, options: argparse.Namespace, scratch_dir: str):
         self.install_guid = str(uuid.uuid4()).upper()
         self.target_guids = {}
         self.version = version
-        self.init_builtins()
+        self.builtins = {} # : Dict[str, UserOption]
```

Comment:
The type here should be `OptionDictType`

Reply:
Need to fix other dict too then. Done.